BPRBE/Persistence/IMongoDependencyRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using BPRBE.Models.Persistence;
+
+namespace BPRBE.Persistence;
+
+public interface IMongoDependencyRepository
```

Comment:
Can probably leave the "Mongo" part out of the interface name. Should be usable for a SQL database too for example.